cmd/meroxa/root/resources/create.go

```diff
@@ -121,11 +123,16 @@ func (c *Create) Execute(ctx context.Context) error {
 		Metadata: nil,
 	}
 
-	if c.flags.Environment != "" {
+	if c.flags.Environment != "" && c.flags.Environment != string(meroxa.EnvironmentTypeCommon) {
```

Comment:
What's the context of this conditional? Could we add a comment or something that could help us understand in the future why these options?

Reply:
comment added